### PR TITLE
Stock span problem python

### PR DESCRIPTION
**Pull request by rutujadhanawade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description
I have added code for Stock span problem in python using Stack.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1566

### CODE REVIEW

**Comment by rutujadhanawade** [comment]
@iamrajiv @plazzy99 Please review this

File: Python/cp/stock_span.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+''' Stock Span Problem
+Given a list of prices of a stock for N number of days, find stock span for each day.
+Stock span is defined as a number of consecutive days prior to the current day when the price of a stock was less than or equal to the price at current day.
+'''
+
+def calcSpan(price):
+    # Create an empty stack
```
Comment (vatsalkesarwani12) [inline Python/cp/stock_span.py]: remove comments from middle of the code

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

File: Python/cp/stock_span.py (hunk 1 above)
Comment (rutujadhanawade) [reply Python/cp/stock_span.py]: Should I remove all the comments from middle of the code? @plazzy99

Comment (vatsalkesarwani12) [reply Python/cp/stock_span.py]: yes

File: Python/cp/stock_span.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+''' Stock Span Problem
+Given a list of prices of a stock for N number of days, find stock span for each day.
```
Comment (iamrajiv) [inline Python/cp/stock_span.py]: @rutujadhanawade Line is too long. It is not following the PEP 8 standard.

File: Python/cp/stock_span.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Stock span is defined as a number of consecutive days prior to the current day when the price of a stock was less than or equal to the price at current day.
+'''
+
+def calcSpan(price):
```
Comment (iamrajiv) [inline Python/cp/stock_span.py:7]: @rutujadhanawade Expected 2 blank lines.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/cp/stock_span.py (hunk 1 above)
Comment (rutujadhanawade) [reply Python/cp/stock_span.py]: done

**Comment by rutujadhanawade** [comment]
I have made all the changes please review. @iamrajiv @plazzy99

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**